docs/API.md

```diff
@@ -8,17 +8,17 @@ A path is added to the Zonemaster GUI base URL, e.g.
 
 The supported paths are the following:
 
-* `check`: the default path, access the input form
-* `domain_check`: same as `check`, but *deprecated*
-* `check/<domain>`: populate the input form with `<domain>`, which is expected
+* `run-test`: the default path, access the input form
+* `domain_check`: same as `run-test`, but *deprecated*
+* `run-test/<domain>`: populate the input form with `<domain>`, which is expected
```

Comment:
run-test/<domain> works, but domain_check/<domain> does not work.

![image](https://user-images.githubusercontent.com/5430733/191734241-4e023c2f-4db2-491e-a505-505f6a2a4018.png)

Reply:
This is the expected behavior (similar to what was done in #334 too).

Reply:
I am not against the behavior, and the document could be read that way, but I suggest that it is clarified that the rewrite is only done when not followed by "/<domain>".

Reply:
I clarified this later in the document.